### PR TITLE
feat: remove anonymous message board user stories

### PR DESCRIPTION
**Pull request by moT01**
Remove user stories/readme from anonymous message board.

Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39721

### CODE REVIEW

File: anonymous-message-board/README.md [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-1. I can report a thread and change its reported value to true by sending a `PUT` request to `/api/threads/{board}` and pass along the `thread_id`. (Text response will be 'success'.)
-1. I can report a reply and change its reported value to true by sending a `PUT` request to `/api/replies/{board}` and pass along the `thread_id` and `reply_id`. (Text response will be 'success')
-1. Complete functional tests that wholely test routes and pass.
-
-|                        | `GET`                      | `POST`                 | `PUT`                  | `DELETE`                              |
-| :--------------------: | :------------------------: | :--------------------: | :--------------------: | :-----------------------------------: |
-| `/api/threads/{board}` | list recent threads        | create thread          | report thread          | delete thread with password           |
-| `/api/replies/{board}` | show all replies on thread | create reply on thread | report reply on thread | change reply to '[deleted]' on thread |
-
```
Comment (RandellDawson) [inline anonymous-message-board/README.md:23]: Do you want to include this table in https://github.com/freeCodeCamp/freeCodeCamp/pull/39652 ?

Comment (moT01) [reply anonymous-message-board/README.md:23]: That PR is specifically to add tests - and it looks like it's pretty much ready to go. So I would say that we should just merge that if it is and make another PR to adjust the description and make sure the user stories are how we want them - and yes, possibly add the table - I would need to look at it more closely to be sure.

**Comment by RandellDawson** [comment]
Are you fine with me merging this PR even though we may want to include the table?  Just don't want to forget about it, since it is not in the normal boilerplate.

**Comment by moT01** [comment]
fine with me - it will remain linked on the issue, so we won't lose it.

**Pull request merged**